openstudiocore/src/model/AirLoopHVAC.cpp

```diff
@@ -914,6 +914,25 @@ namespace detail {
     bool result = setString(OS_AirLoopHVACFields::DesignSupplyAirFlowRate, "autosize");
     OS_ASSERT(result);
   }
+  
+  /** Baseline Pressure Drop Adjustment **/
+  // Return the Baseline Pdrop
+  double AirLoopHVAC_Impl::baselinePressureDropAdjustment() const {
+    return getDouble(OS_AirLoopHVACFields::BaselinePressureDropAdjustment,true);
+  }
+  
+  // Set the Baseline Pdrop
+  bool AirLoopHVAC_Impl::setBaselinePressureDropAdjustment(double baselinePressureDropAdjustment) {
+    result = setDouble(OS_AirLoopHVACFields::BaselinePressureDropAdjustment, baselinePressureDropAdjustment);
+    OS_ASSERT(result);
+  }
+  
+  // Reset Baseline Pdrop
+  void AirLoopHVAC_Impl::resetBaselinePressureDropAdjustment() {
+    bool result = setDouble(OS_AirLoopHVACFields::BaselinePressureDropAdjustment, 0);
```

Comment:
No I agree with you here @macumber. Reset to empty field so you are back to the state you constructed at.